integrations/google_genai/src/haystack_integrations/components/generators/google_genai/chat/chat_generator.py

```diff
@@ -580,12 +627,14 @@ def run(
             if streaming_callback:
                 # Use streaming
                 response_stream = self._client.models.generate_content_stream(
-                    model=self._model, contents=contents, config=config
+                    model=self._model, contents=cast(Any, contents), config=config
```

Comment:
It seems that `cast(Any)` is very similar to `type:ignore`. Why do we need this?

If possible, I would prefer to fix typing issues.
If not possible, we can use `type:ignore[specific-error-ignored]` with a comment explaining why we need this.